msg/vehicle_rates_setpoint.msg

```diff
@@ -1,8 +1,12 @@
 uint64 timestamp	# time since system start (microseconds)
 
-float32 roll		# body angular rates in NED frame
-float32 pitch		# body angular rates in NED frame
-float32 yaw		# body angular rates in NED frame
+# body angular rates in NED frame
+float32 roll		# [rad/s] roll rate setpoint
+float32 pitch		# [rad/s] pitch rate setpoint
+float32 yaw		# [rad/s] yaw rate setpoint
+
+# [rad/s] yaw rate setpoint the wheel attempts to track (the wheel controller has different gains than the yaw controller)
+float32 yaw_via_wheel
```

Comment:
Is there actually a case where the rates setpoints differ what the wheel is trying to track vs what the vehicle is trying to track?

Reply:
yeah I don't really like this either -- originally intended to try and clean this up a bit.. open to recommendations. the issue here is that the wheel and yaw rate controllers are separate. with different tunings. currently the rudder signal takes the output of one or the other depending on the mode. i guess maybe could be cleaner here to just populate this setpoint with the one currently being used, but then we also lose some information.

ideally would be nice to have them as multiple inputs to control yaw rate with control allocation effectiveness modeled appropriately for each while on ground or not... but this requires some more work to better detect "on ground".